packages/manager/apps/web/client/app/email-domain/order/EmailDomainOffer.class.js

```diff
@@ -0,0 +1,20 @@
+import find from 'lodash/find';
+import get from 'lodash/get';
+import includes from 'lodash/includes';
+
+export default class EmailDomainOffer {
+  constructor({
+    planCode, prices, productName, productType,
+  }) {
+    Object.assign(this, {
+      planCode,
+      prices,
+      productName,
+      productType,
+    });
+  }
+
+  getPrice(capacity) {
+    return get(find(this.prices, ({ capacities }) => includes(capacities, capacity)), 'priceInUcents');
```

Comment:
This isn't handled by the Cart handling library ?

Reply:
There is no Cart handling library, just a service 😄 
What do you think about a super class that handles it and EmailDomain inherit from it ?